website/docs/components/button/partials/guidelines/guidelines.md

```diff
@@ -84,6 +84,44 @@ Buttons used for a Dropdown (with the chevron icon) can be found in [Dropdown](/
   <Hds::Button @color="secondary" @text="No text" @icon="plus" @isIconOnly={{true}} />
 </Hds::ButtonSet>
 
+### Leading
+
+In most cases, use leading icons. Choose icons that add meaning and clarity to the action described in the link’s text.
+
+<Hds::ButtonSet>
+  <Hds::Button @color="secondary" @text="Unlock" @icon="unlock" @iconPosition="leading" />
+  <Hds::Button @color="secondary" @text="Show more results" @icon="eye" @iconPosition="leading" />
```

Comment:
If a consumer came to me with the "Show more results" example, I'd recommend they remove the icon completely because it doesn't provide additional value and increases the length of the button unnecessarily. For that reason, I'd remove it as an example.

Reply:
+1

Reply:
Makes sense. I removed it from the examples.